dcos/subcommand.py

```diff
@@ -141,7 +141,7 @@ def default_subcommands():
     :returns: list of all the default dcos cli subcommands
     :rtype: [str]
     """
-    return ["auth", "config", "help", "marathon",
+    return ["auth", "config", "help", "job", "marathon",
```

Comment:
ok.. I read the comment... I have no idea what it means.  any search for `default_subcommands` results in this comment only and main.py.  looking...